.github/ISSUE_TEMPLATE/new-feature.yml

```diff
@@ -0,0 +1,55 @@
+name: New Content
+description: For new features or content suggestions
+title: '[NEW]: '
+labels: ['new content']
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to submit this issue. Your input is really appreciated.
+  - type: dropdown
+    id: issue-priority
```

Comment:
I don't think we should have a priority option, since not all issue raisers will know, and some might differ in opinion from other stakeholders.